Engage: Add more articles related to Upgrades (#6774)
umbraco/UmbracoDocs: 13/umbraco-engage/release-notes.md

---
description: Get an overview of the changes and fixes in each version of Umbraco Engage.
---

# Release Notes

> In this section, you can find a summary of changes to Umbraco Forms released in each version. Each version is presented with a link to the [Engage issue tracker](https://github.com/umbraco/Umbraco.Engage.Issues/) showing a list of issues resolved in the release. We also link to the individual issues themselves from the detail.

Reviewer: ```suggestion
In this section, you will find a summary of changes made to Umbraco Engage that were released in each version. Each version has a link to the [Engage issue tracker](https://github.com/umbraco/Umbraco.Engage.Issues/) showing a list of issues resolved in the release. We also link to the individual issues themselves from the detail.
```

If there are any breaking changes or other issues to be aware of when upgrading they are also noted here.

{% hint style="info" %}

> When upgrading to a major version, learn about the breaking changes in the [Version Specific Upgrade Notes](upgrading/version-specific-upgrade-notes.md) article.

Reviewer: ```suggestion
When upgrading to a major version, be sure to look at the breaking changes outlined in the [Version Specific Upgrade Notes](upgrading/version-specific-upgrade-notes.md) article.
```
